libs/pxweb2-ui/src/lib/components/Select/Select.tsx

```diff
@@ -0,0 +1,164 @@
+import cl from 'clsx';
+import classes from './Select.module.scss';
```

Comment:
Can we add a blank space between imports of the external packages and the internal files? Makes it easier to parse them for me atleast. So between the current lines 1 and 2.